src/alfalfa/AlfalfaActuator.hpp

```diff
@@ -1,3 +1,8 @@
+/***********************************************************************************************************************
+*  OpenStudio(R), Copyright (c) Alliance for Sustainable Energy, LLC.
+*  See also https://openstudio.net/license
+***********************************************************************************************************************/
```

Comment:
I used `ruby developer/ruby/ApplyCopyright.rb` which is responsible for most of these changes